docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+  companies would like to rely on Kubernetes state-based management but they need more flexibility to accomodate for conditions like
+   large-scale deployments, disconnected and autonomous operation, and constrained cluster size. These requirements represent the Edge 
+   Multi-cluster use case.
+
+A key area of investigation for `kcp` is exploring large-scale deployment of workloads that can operate under constraints and
+ conditions unlike those accomodated by upstream Kubernetes. Ideally, we want declaritively defined workloads to operate across a
+  hierarchy combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. Deployment
+   of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+    status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
+
+There are many different ways to characterize deployments of workloads in the Edge use case. Support for blue/green, canary, carbon
+ intensity sensitive, and threat avoidance are some of the deployment strategies to consider when casting out 100's or 1000's of
+  workloads to millions of destinations. Having the ability to assign workloads using a declaritively defined placement strategy to
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, integrations, and security
+
+Today's smartphone is not only the property of the user who purchased it. The reality is that application developers (app store),
+ corporate entities (vpn profiles), and the smartphone manufacturer have roles and responsibilities on these devices. Edge cluster
+  locations are no different in this respect. A scalable Edge multi-cluster solution can thrive and advance if there is support for
+   community contribution powered by integration and proper security features. All of these considerations must work across an evolving
+    hierarchy that combines small and large control plane API services.
+
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of transparent multi-cluster which can be used to expand the number of entry points for workloads and
+ operations by reducing the size of the control plane to a single binary. Synchronization between KCP virtual workspaces and physical
+  clusters has made it possible to deploy workloads centrally and then fan out to any defined subset of clusters. We would like to
+   leverage multiple levels of hierarchy (cascading multiple instances of KCP and Kubernetes cluster) to arrive at a distributed system
+    capable of reaching any targeted edge cluster with/without edge devices attached. Building out such a hierarchy will require a
```

Comment:
Not sure the scale should be linked to the hierarchical KCP architecture. Scale need to be explored because the large scale nature (millions of targets) of the use case. Actually hierarchical architecture may be a possible solution for some of the scaling issues (including the DB if we can avoid centralized place for data)

Reply:
Yes.
Remember also how scale complicates other aspects of the problem, making simple solutions infeasible.

Reply:
I remove 'hierarchy' and replaced with 'flexible topology'